### PR TITLE
Update ONT flowcells page

### PR DESCRIPTION
**Pull request by kedhammar**
Reorganize columns, parse project information and add lots of hrefs to LIMS.

Make Orlando very happy

### CODE REVIEW

File: run_dir/design/ont_flowcells.html [hunk 1]
```diff
@@ -47,13 +55,40 @@ <h1 class="ont_fc_page_heading" id="ont_page_title">ONT Flowcells</h1>
                 <abbr title="Ongoing"><span class="fa fa-hourglass-half"></span></abbr>
             {% end %}
         </td>
-        <td>{{ ont_flowcells[onefc]["start_date"] }}</td>
+
+
+        {% if find_id(ont_flowcells[onefc]['sample_name'], 'sample') %}
+            <td class="sample">
+                <a class="text-decoration-none" href="https://ngi-lims-prod.scilifelab.se/clarity/search?scope=Sample&query={{ find_id(ont_flowcells[onefc]['sample_name'], 'sample') }}">
```
Comment (alneberg) [inline run_dir/design/ont_flowcells.html:62]: Bonus points if this link points to lims-stage if genstat is running from statusdb dev...

File: status/flowcell.py [hunk 2]
```diff
@@ -220,7 +220,12 @@ def fetch_ont_run_stats(view_all, view_project, run_name):
 
     # Try to find project name. ID string should be present in MinKNOW field "experiment name" by convention
     query = re.compile("(p|P)\d{5}")
```
Comment (alneberg) [inline status/flowcell.py:222]: Might as well have 5 or 6 digits for future proofing?

File: status/flowcells.py [hunk 3]
```diff
@@ -40,6 +41,26 @@ def formatDate(date):
...
     return datestr.strftime("%a %b %d %Y, %I:%M:%S %p")
 
+
+def find_id(stringable, pattern_type: str) -> re.match:
+
+    string = str(stringable)
+
+    patterns = {
+        "project": re.compile("P\d{5}"),
+        "sample": re.compile("P\d{5}_\d{3}"),
```
Comment (alneberg) [inline status/flowcells.py:51]: Throw in possibility of 6 digit projects here as well and we'll be safer for the future.

**Review by alneberg — APPROVED** [review]
Looks good! Just a minor change that I wish for.

File: status/flowcells.py (hunk 3 above)
Comment (kedhammar) [reply status/flowcells.py]: [93b885a](https://github.com/SciLifeLab/genomics-status/pull/740/commits/93b885adb1c6aa6b7eccb8759d5b8da9a4745a6d)

File: status/flowcell.py (hunk 2 above)
Comment (kedhammar) [reply status/flowcell.py]: [93b885a](https://github.com/SciLifeLab/genomics-status/pull/740/commits/93b885adb1c6aa6b7eccb8759d5b8da9a4745a6d)

File: run_dir/design/ont_flowcells.html (hunk 1 above)
Comment (kedhammar) [reply run_dir/design/ont_flowcells.html]: [93b885a](https://github.com/SciLifeLab/genomics-status/pull/740/commits/93b885adb1c6aa6b7eccb8759d5b8da9a4745a6d)

**Pull request merged**